src/vueWrapper.ts

```diff
@@ -13,6 +13,7 @@ import { TriggerOptions } from './createDomEvent'
 import { find } from './utils/find'
 import { isFunctionalComponent } from './utils'
 
+type NonexistentDomWrapper<T extends Element> = Omit<DOMWrapper<T>, 'exists'>
```

Comment:
nit: the name `NonexistentDomWrapper` sounds a bit weird to me. To my french ear, it sounds like the DomWrapper does not exist 😅 I think we should name it differently, but I don't have a great suggestion...

Reply:
I had the same thought... if anything this should `ExistsDomWrapper` or `NotNullableDomWrapper`.

Since this type is not used much, could we just inline it?

```
get<K extends keyof HTMLElementTagNameMap>(
  selector: K
):    ): Omit<<HTMLElementTagNameMap[K]>, 'exists'>
```

^ Probably wrong but that would be the idea.